shapely/speedups.py

```diff
@@ -1,3 +1,5 @@
+"""Speedups for Shapely geometry operations."""
```

Comment:
It could be worthwhile mentioning at the top that this module will be removed in a future release.